spec/requests/bulk_export_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'spec_helper'
+
+describe BulkExportController do
+  let!(:admin) { create(:admin) }
+  let!(:regular_user) { create(:user) }
+  let!(:collection) { create(:collection, owner_user_id: admin.id) }
+  let!(:bulk_export) { create(:bulk_export, collection: collection, user: admin) }
+
+  describe '#index' do
+    let(:action_path) { bulk_export_index_path }
+    let(:subject) { get action_path }
+
+    it 'redirects when not logged in' do
+      subject
+      expect(response).to have_http_status(:redirect)
+      expect(response).to redirect_to(dashboard_path)
+    end
+
+    it 'redirects when logged in as regular user' do
+      login_as regular_user
+      subject
+      expect(response).to have_http_status(:redirect)
+      expect(response).to redirect_to(dashboard_path)
+    end
+
+    it 'renders when logged in as admin' do
+      login_as admin
+      subject
+      expect(response).to have_http_status(:ok)
+      expect(response).to render_template(:index)
+    end
+  end
+
+  describe '#show' do
+    let(:action_path) { bulk_export_show_path(bulk_export_id: bulk_export.id) }
+    let(:subject) { get action_path }
+
+    it 'redirects when not logged in' do
+      subject
+      expect(response).to have_http_status(:redirect)
+      expect(response).to redirect_to(dashboard_path)
+    end
+
+    it 'redirects when logged in as regular user' do
+      login_as regular_user
+      subject
+      expect(response).to have_http_status(:redirect)
+      expect(response).to redirect_to(dashboard_path)
+    end
+
```

Comment:
Let's add 2 tests for the collection owner user as well.  If they own (or are staff) on the collection, then they should be able to see the bulk_export_show_path (I think -- where is this linked from?).  If they don't own it, they shouldn't be able to see it.